elasticsearch/examples/6.x/Makefile

```
@@ -13,3 +13,4 @@ test: install goss
 
 purge:
 	helm del --purge $(RELEASE)
+	for pvc in $$(kubectl get pvc -o name | grep $(RELEASE)); do kubectl delete $$pvc; done
```

I think the `pvc` part is missing here.
```suggestion
	for pvc in $$(kubectl get pvc -o name | grep $(RELEASE)); do kubectl delete pvc $$pvc; done
```

You could also drop the for loop and just do `kubectl delete pvc $$(kubectl get pvc -o name | grep $(RELEASE))`

---

>I think the pvc part is missing here.

Actually with the `-o name`, k8s resources are already prefixed by their kind:
```
$ kubectl get pvc -o name | grep helm-logstash-oss
persistentvolumeclaim/helm-logstash-oss-logstash-helm-logstash-oss-logstash-0
```

---

>You could also drop the for loop and just do `kubectl delete pvc $$(kubectl get pvc -o name | grep $(RELEASE))`

Nice I didn't know `kubectl delete` could accept more than one resource in argument, I prefer avoiding the `for` loop if needed 👍

---

fixed in https://github.com/elastic/helm-charts/pull/336/commits/1e5616b3a8050b4810a2ff4974a01d22c207aab1